include/spatio_temporal_voxel_layer/spatio_temporal_voxel_grid.hpp

```diff
@@ -134,12 +135,9 @@ class SpatioTemporalVoxelGrid
 
   // Check occupancy status of the grid
   bool IsGridEmpty(void) const;
-
-  // Get time information for clearing
-  double GetDecayTime(void);
-  double GetAcceleratedDecayTime(const double& acceleration_factor);
-
-  // Clearing grid operation with frustum models
+  double GetDecayShift(const double& time_delta);
```

Comment:
I had a comment grouped here for "Get time information for clearing", is there a reason you removed it?

Reply:
you didnt add back in this comment